### DISCUSSION TITLE
Details for 3D Printed Parts

### DISCUSSION THREAD

**Discussion by brightsnowzhao** [prl-mushr/mushr#60] [Hardware]
Hi, is there an instruction on recommended 3D printing parameters?  I'm looking for the proper layer height and infill rate.  I tried 0.2mm  and 15% to print the servo cage.  It looks OK except that the nut holes appear to be too loose and the nut can rotate inside.  I had to use a pin to hold it in order to tighten.

**Comment by brightsnowzhao** [comment]
Quick update - I've tried a few settings and found that 0.2mm with 15% to 20% filling is good.  The parts are solid enough and fine enough.  It appears to be related to the quality of the printer.  I start to get worried about the quality when the layer height was increased to 0.3mm.  I'm printing all the parts at 0.2mm now.

**Comment by palanc** [comment]
Hi @brightsnowzhao  - Sorry for not replying sooner, and thank you for sharing settings that have worked for you! We've had success with similar settings (0.2mm layer height, 20-25% infill).
